lib/template-loader.php

```diff
@@ -192,9 +192,8 @@ function gutenberg_find_template( $template_file ) {
 				(object) $current_template_post
 			);
 		}
-
-		$_wp_current_template_content = $current_template_post->post_content;
 	}
+	$_wp_current_template_content = $current_template_post->post_content;
```

Comment:
`$current_template_post` can be `null` in this scope, causing an error.

![image](https://user-images.githubusercontent.com/1779930/69999405-97539800-1526-11ea-8cc8-96e85700bd09.png)

Fairly simple to reproduce in stock install with TwentyTwenty, Gutenberg, only "Enable Full Site Editing" experiment enabled (not the demo templates, which I assume would allow a default to be set).

Reply:
https://github.com/WordPress/gutenberg/pull/18877/commits/2f18a4aeb3a31d8798f028a742755109a558176b